### DISCUSSION TITLE
Cut/Delete a portion of multiple images

### DISCUSSION THREAD

**Discussion by AKAKillroy** [ImageMagick/ImageMagick#5626] [Help]
I am building an arcade cabinet / machine and I have a bunch of (same size) images that contain the arcade machine "bezel" or surround as it would be in the original 80s-90's arcade machine but the images I have need to to be cut out as they are too large (inner image) and they cut off the play area. Simply put I have several hundred like images that I want to "cut out" the inside of the image to a specific size and leave the rest of the image in tact. 

IMAGE
+---------------+
|    +--------+    |
|     |   CUT   |     |
|     |             |     |
|    +--------+    |
+---------------+

**Comment by fmw42** [comment]
Please post an actual image example and not a diagram. We need to test with your image and also to better understand your issue.

Also please identify your IM version and platform/OS
